conjure-codegen/src/human_size.rs

```diff
@@ -0,0 +1,50 @@
+pub fn parse(s: &str) -> Result<usize, String> {
```

Comment:
Is this based on an equivalent java class where we are trying to maintain the same logic? Or is there some crate we could use that handles this so we're not worried about edge cases?

The logic here seems fine to be but I'm a bit paranoid that there's some edge case we've missed.

Reply:
Yeah, the Java version is here: https://github.com/palantir/human-readable-types/blob/develop/human-readable-types/src/main/java/com/palantir/humanreadabletypes/HumanReadableByteCount.java. It should be the same except the Rust version doesn't currently handle `byte`, `mebibyte`, etc since I never see those used in practice.

Reply:
I haven't found an existing crate that handles this parsing unfortunately.